app/routes/pages/components/subcomponents/TextWithTitle/index.tsx

```diff
@@ -1,24 +1,31 @@
+import cx from 'classnames';
 import styles from './TextWithTitle.css';
 import type { ReactNode } from 'react';
 
 type Props = {
   title: string;
   text: ReactNode;
   extraStyle?: Record<string, any>;
+  extraClassName?: string;
 };
 
-const TextWithBoldTitle = ({ title, text, extraStyle }: Props) => {
+const TextWithBoldTitle = ({
+  title,
+  text,
+  extraStyle,
+  extraClassName,
+}: Props) => {
   return (
-    <div className={styles.container} style={extraStyle}>
+    <div className={cx(styles.container, extraClassName)} style={extraStyle}>
       <h2 className={styles.boldTitle}>{title}</h2>
       <p className={styles.text}>{text}</p>
     </div>
   );
 };
 
-const TextWithTitle = ({ title, text, extraStyle }: Props) => {
+const TextWithTitle = ({ title, text, extraStyle, extraClassName }: Props) => {
   return (
-    <div className={styles.container} style={extraStyle}>
+    <div className={`${styles.container} ${extraClassName}`} style={extraStyle}>
```

Comment:
`cx`?

Reply:
Ah yes, was testing moving back and forth to see if the css selectors specificity would change (it didn't:/) - moved back to cx now